docs/ref/forms/api.txt

```diff
@@ -1106,10 +1123,16 @@ they're not the only way a form object can be displayed.
 
 .. class:: BoundField
 
-   Used to display HTML or access attributes for a single field of a
-   :class:`Form` instance.
+    Used to display HTML or access attributes for a single field of a
+    :class:`Form` instance.
+
+    The ``__str__()`` method of this object displays the HTML for this field.
+
+    You can use :attr:`.Form.bound_field_class` and
+    :attr:`.Field.bound_field_class` to use a different ``BoundField`` class
+    per-form or per-field.
```

Comment:
```suggestion
   Used to display HTML or access attributes for a single field of a
   :class:`Form` instance.

   The ``__str__()`` method of this object displays the HTML for this field.

   You can use :attr:`.Form.bound_field_class` and
   :attr:`.Field.bound_field_class` to use a different ``BoundField`` class
   per-form or per-field.
```
(nit) this has been indented slightly off

Reply:
What? 3 spaces instead of 4? I don't understand: it's not consistant with the rest of the documentation, is it?

Reply:
Ah you're right, it was indented wrong previously and you have fixed it :+1: